### ISSUE TITLE
turn into a proper python package?

### ISSUE THREAD

**Issue by xrotwang** [ZAS-QUEST/eldpy#17]
Are there any plans to turn into this into a "proper" python package (i.e. installable via pip, preferably from pypi, ...)?

At least in case community collaboration is intended, proper python project structure would help a lot with introducing new people to the functionality.

**Comment by Glottotopia** [comment]
yes, this is on the todolist, but I have to be reasonably confident in the backend data structure before I start doing this. One part is the retrieval, which is currently found in `bulkdownload.py`. That part is pretty isolated and easy to grasp. The other part is the analysis of the elan files, where there will be a lot of changes and which is not ready for production.
